canonical-web-and-design/charmhub.io
# Add JS tests for store

## Done

- Setup JS testing infrastructure
- Add tests for JS

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run `dotrun test-js` and see all tests pass


## Issue / Card

Fixes #205 

## Screenshots

[if relevant, include a screenshot]
